packages/dart/npt_flutter/lib/features/policy/cubit/policy_cubit.dart

```diff
@@ -0,0 +1,318 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../repositories/role_repository.dart';
+import '../models/policy.dart';
+
+part 'policy_state.dart';
+
+class PolicyCubit extends Cubit<PolicyState> {
```

Comment:
Extend the LoggableCubit instead to make sure Policy Events are (temporarily) logged in the app's (in memory) debug logs